projects/plugins/crm/admin/contact/view.page.php

```diff
@@ -872,7 +869,7 @@ function jpcrm_render_contact_view_page( $id = -1 ) {
 
 							// prep link to create a new invoice
 							$new_invoice_url = jpcrm_esc_link( 'create', -1, ZBS_TYPE_INVOICE ) . '&zbsprefillcust=' . $contact['id'];
-							if ( $contact_invoice_count > 0 ) {
+							if ( count( $contact['invoices'] ) > 0 ) {
```

Comment:
`$contact_invoice_count` still exists if you want to use it.

Reply:
I believe that's the same situation as above: https://github.com/Automattic/jetpack/pull/30178#discussion_r1182581342 (we want the real, full count).

Reply:
Got it, makes sense to use the new key!